mesheryctl/internal/cli/root/system/start.go

```diff
@@ -102,8 +102,15 @@ func start() error {
 		return err
 	}
 
-	if utils.PlatformFlag != "" {
-		currCtx.SetPlatform(utils.PlatformFlag)
+	if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
+		currCtx.Platform = utils.PlatformFlag
+		err := utils.ChangePlatform(mctlCfg.CurrentContext, currCtx)
+
+		if err != nil {
+			return err
+		}
+	} else {
+		return errors.New(fmt.Sprintf("the platform %s is not supported currently. The supported platforms are:\ndocker\nkubernetes\nPlease check %s/config.yaml file.", utils.PlatformFlag, utils.MesheryFolder))
```

Comment:
Looks good for the most past @nehaLohia27 !

@piyushsingariya should the location of the config file be hardcoded to the default? I think the message should display the location of the config file when the user changes it using the flag too...
What do you think?

Reply:
right! @nehaLohia27 can you implement this?

Reply:
@piyushsingariya @metonymic-smokey i have implemented the code. But PR keeps failing. not sure what the problem. can you please help ?

Reply:
Great catch, @metonymic-smokey